client/app/scripts/liveblog-edit/directives.js

```diff
@@ -253,6 +256,26 @@ define([
                                     notify.error(gettext('Something went wrong. Please try again later'));
                                 });
                             },
+                            highlightPost: function(post) {
+                                scope.clearReorder();
+                                if (post.highlight == false) {
```

Comment:
please remove this is is not needed.
you can just use two messages base on the highlight.
ex: notify.info( post.highlight? gettext('Post was highlighted') ...